fanuc_lrmate200id_moveit_plugins/CMakeLists.txt

```diff
@@ -24,6 +26,8 @@ catkin_package(
     pluginlib
     roscpp
     tf_conversions
+  DEPENDS
+    urdfdom_headers
```

Comment:
Is this really necessary? Doesn't the `depend` on `moveit_core` already pull this in ([here](https://github.com/ros-planning/moveit/blob/e6d0f3442c39e2b6948f443ccd2e93be190c4a61/moveit_core/package.xml#L33))?

Similar comment for the other `urdfdom_headers` related statements.